Various technical notes/transition_infos.json

```diff
@@ -803,11 +786,6 @@
           "area_id": "0x8147EA91",
           "area_name": "Copacanti Lake",
           "requires": []
-        },
-        {
-          "area_id": "0x5511C46C",
-          "area_name": "Apu Illapu Shrine (Spinjas)",
-          "requires": []
         }
```

Comment:
Why remove this?

```suggestion
        },
        {
          "area_id": "0x5511C46C",
          "area_name": "Apu Illapu Shrine (Spinjas)",
          "requires": []
        }
```

Reply:
This is one of the one-way-exits. All one-way-exits are removed from the .json file and instead handled in the entrance_rando.py file.

Reply:
All 4 one-way exits are put back in the json file, but randomizer is not affected.